_maps/map_files/SeleneStation/SeleneStation.dmm

```diff
@@ -24603,10 +24604,8 @@
 	pixel_x = -3;
 	pixel_y = 1
 	},
-/obj/item/storage/box/bodybags,
```

Comment:
Why remove these bodybags?

Reply:
I don't remember every deleting that wut